src/nft/utils/listNfts.ts

```diff
@@ -37,11 +37,12 @@ export const ListingMarkets: ListingMarket[] = [
     fee: 0.5,
     icon: '/nft/svgs/marketplaces/x2y2.svg',
   },
-  {
-    name: 'LooksRare',
-    fee: 1.5,
-    icon: '/nft/svgs/marketplaces/looksrare.svg',
-  },
+  // Hiding LooksRare until we can support their v2 SDK
+  // {
+  //   name: 'LooksRare',
+  //   fee: 1.5,
```

Comment:
is the fee for v2 going to be `1.5` also?

Reply:
No it would be 0.5% but I'm keeping all updates to the LR listing logic for when we actually implement v2